lib/book.rb

```diff
@@ -0,0 +1,18 @@
+class Book
+  attr_reader :author_first_name, :author_last_name, :title, :publication_date
+
+  def initialize(book_info)
+    @author_first_name = book_info[:author_first_name]
+    @author_last_name = book_info[:author_last_name]
+    @title = book_info[:title]
+    @publication_date = book_info[:publication_date]
+  end
+
+  def author
+    [@author_first_name, @author_last_name].join(" ")
```

Comment:
interesting way to do this. Ruby allows us to "add" strings together, so `@first_name + ' ' + @last_name` would be simpler